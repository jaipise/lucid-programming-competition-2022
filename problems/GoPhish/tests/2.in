20
NBMlwsNjdLaQBueFWvUXqEw
TtfhlIXSWHxVG ALZDqKreVrj
xZdtBYZArFvTQcPBfs
cWgZhr
IvzjFGgKAnrrkjCaKEM
VzQRAbRdsphrgZtMbmPyvEfp
mcznpDvf
wzer
ttvZfVSUxFGOrnfSlxcJhsrCX
rXKNkXAPbxrRftKs
UqPLdhguaCpo
EYyIrWbVuFceNGclLvsYzl
GzjJLcFuZSIshJ JNzsSOamRL
dAdxJeKqTJgVcVABXBJHgJRjRPY
kId J
PbAUfcbKexqZbMmgHenvNzsh
ZMkcUNbALFhMtPmQrsoMQUpVp
sNaDWHBSakHWIRnfOUpalvdxH
csRtatYESbondMdnLyq
sUNASffWGZ
40
VzQRAbRdsphrgZtMbmPyvEfp
uQplDHGUAcPO
gZJjlCfUzsiSHj jnZSsoAMrl
QnliaVNpLkzRbefzSk
TtfhlIXSWHxVG ALZDqKreVrj
hWZER
XmJXUQWCqNHeSzNJzTRak pKuy
UMEKRq
uVTcnYCMclE
zmKCunBalfHmTpMqeRSOmquPvP
RxknKxapBRrFTkS
vZqraBrDSPHRGzTmBMpYVeFP
zmKCunBalfHmTpMqRSOmquPvP
jBWwNEqRGLqetm a
ZMkcUNbALFhMtPmQrsoMQUpVp
NBMlwsNjdLaQBueFWvUXqEw
sUNOASffWGZ
oE uPqFMK
ob TtGIhWGRkQgxDOQHNoaF
XzDTbyzaRfVtqCpbFS
LFNaMjJDGQavW
oMbeNDgaMyfe
vZqrFaBrDSPHRGzTmBMpYVeFP
ttvZfVSUxFGOrnfSlxcJhsrCX
csWgZhr
EYyIrWbVuFceNGclLvsYzl
asAFSsgRC
PbAUfcbKexqZbMmgHenvNzsh
MCZNPdVjF
zgdxRRPO
TtfhlIXSWHxVG ALZDqKreVrj
PbAUfcbKexqZbMmgHenvNzsh
sUNALWZ
pBauFCBkEXQzBmMhENVnZSH
EYyIrWbVuFceNGclLvsYzl
RxknKxapBXRrFkS
iRRtZcJfVswysBzUcBOb
YmltXVE PBFyEBw
Uv ZFQGglM
TTVzFvsuXfgoRNFsLXCjHSRcx
