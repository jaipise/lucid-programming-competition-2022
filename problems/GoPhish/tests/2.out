EMPLOYEE
EMPLOYEE
EMPLOYEE
CUSTOMER
EMPLOYEE
BOT
CUSTOMER
CUSTOMER
CUSTOMER
BOT
BOT
EMPLOYEE
EMPLOYEE
CUSTOMER
EMPLOYEE
EMPLOYEE
BOT
CUSTOMER
CUSTOMER
EMPLOYEE
CUSTOMER
CUSTOMER
BOT
EMPLOYEE
BOT
EMPLOYEE
CUSTOMER
EMPLOYEE
BOT
CUSTOMER
EMPLOYEE
EMPLOYEE
CUSTOMER
BOT
EMPLOYEE
BOT
CUSTOMER
CUSTOMER
CUSTOMER
EMPLOYEE
